tests/test_s3.py

```diff
@@ -742,7 +757,7 @@ def test_bucket_in_other_region(self):
 
     @run_until_complete
     def test_bucket_in_other_region_using_http(self):
-        client = self.session.create_client(
+        client = self.create_client(
```

Comment:
Not sure if this works, should produce `RuntimeError`, since you do `loop.run_until_complete` inside coroutine, ``yield from create_client(....)`` will work or ``yield from self.session.create_client(....)``